src/registrar/forms/domain.py

```diff
@@ -228,12 +232,16 @@ class DomainDnssecForm(forms.Form):
 class DomainDsdataForm(forms.Form):
     """Form for adding or editing DNSSEC DS Data to a domain."""
 
+    def validate_hexadecimal(value):
+        if not re.match(r"^[0-9a-fA-F]+$", value):
```

Comment:
(Optional) The function name is generally pretty descriptive, but a comment here or function description would be a nice to have that this regex specifically matches a string that consists entirely of one or more hexadecimal digits.

Reply:
added a comment